src/Message/Handler/AssignEligibleCalloutsToProductHandler.php

```diff
@@ -19,17 +18,12 @@ final class AssignEligibleCalloutsToProductHandler implements MessageHandlerInte
     /** @var ProductRepositoryInterface */
     private $productRepository;
 
-    /** @var ObjectManager */
-    private $productManager;
-
     public function __construct(
         EligibleCalloutsProviderInterface $eligibleCalloutsProvider,
-        ProductRepositoryInterface $productRepository,
-        ObjectManager $productManager
```

Comment:
This change essentially makes this is a breaking change, but I guess that's okay

Reply:
It's not yet stable, so it's ok.